### PR TITLE
Feat/event page

### PR DESCRIPTION
**Pull request by MichaelBLemaireJr**
Now clicking on events will send you to a new page and clicking the home button sends you to the main page

### CODE REVIEW

File: react-client/src/components/navbar.jsx [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-const Navbar = () => (
-  <h2>Navbar</h2>
+// eslint-disable-next-line react/prop-types
+const Navbar = ({ setClickEventBackToNull }) => (
+  <div>
+    <h2>Navbar</h2>
+    <button type="button" onClick={setClickEventBackToNull}>Home</button>
```
Comment (alexa-w) [inline react-client/src/components/navbar.jsx:7]: Do we want this to be in state or do we want it to be a plain old GET / ? Either way is cool

**Review by alexa-w — COMMENTED** [review]
yeah, this is pretty sick. I have to ask: when are we making network requests and when are we checking authentication? Will be have a `loggedIn` flag in global state determining permissions?

File: react-client/src/components/navbar.jsx (hunk 1 above)
Comment (alexa-w) [reply react-client/src/components/navbar.jsx:7]: I'm trying to wrap my head around exactly what we should expect when it comes to making requests to the server. We have to be prepared for the possibility that we just can't load all of our events at once with the page. I think most of this is fine as is, here -- the main authentication barriers are event creation and rsvp'ing, and the user "dashboard"

Comment (alexa-w) [reply react-client/src/components/navbar.jsx:7]: Do you think we should just make new network requests for events when a category is selected?

Comment (cskladz) [reply react-client/src/components/navbar.jsx:7]: we could filter out the events by category and assign them to the state, when someone clicks on "table-top" the value of table-top on the state would have the data to render the list items...

**Review by cskladz — COMMENTED** [review]
I can work with this, I would be plugging in my <Map /> component on CurrentlyClickedEvent, i can play around with the positioning too, also should we be thinking about css tags such as id and class for these html elements we're rendering in jsx?

**Comment by alexa-w** [comment]
Honestly, I feel like it would be disingenuous to have all of the events on
the client at once -- rather, the server would make a filtered query by
category to the DB. This would be triggered by selecting a category. Also
it's fine to worry about css a little in a bit, but let's get that MVP
first imo

**Comment by cskladz** [comment]
Yeah sounds good, have one state property responsible for holding the data rendered by the list, and just have the click event require the server to supply a new data set thats been filtered appropriately for the requested category

**Comment by cskladz** [comment]
Im gonna merge this so i can pull it down and add a map feature in the appropriate place

**Pull request merged**